salt/orchestrate/aws/map_templates/instance_map.yml

```diff
@@ -10,10 +12,12 @@
       tag:
         environment: {{ environment_name }}
         role: {{ service_name }}
+        created_at: {{ salt.status.time(format=ISO8601) }}
```

Comment:
It is redundant for EC2, but most other resources are lacking that. This is just to create a consistent set of values that we can refer to without having to look in different places for each type of asset.